source/refined-github.css

```diff
@@ -1,3 +1,26 @@
+@import './refined-github.css';
```

Comment:
Oops. Recursive import

I also need to think of a way to extract the raw CSS below into another file because this change changes the cascade order.